src/nvim/api/win_config.c

```diff
@@ -490,7 +495,7 @@ void nvim_win_set_config(Window window, Dict(float_config) *config, Error *err)
       win->w_pos_changed = true;
     }
 
-    int flags = win_split_flags(fconfig.split, parent == NULL);
+    int flags = win_split_flags(fconfig.split, parent == NULL) | WSP_NOENTER;
```

Comment:
The only uses of `win_split_flags` add `| WSP_NOENTER`. Maybe we should just move that into the function?

Reply:
Subjective (especially as `win_split_flags` is static too), but I think I prefer `WSP_NOENTER` being set at the caller's discretion.